### PR TITLE
Feature vite setup basic

### PR DESCRIPTION
**Pull request by AloofBuddha**
Setup basic frontend scaffolding with Vite

Ticket: https://trello.com/c/a8Xku8oX

### CODE REVIEW

**Comment by ivangoudyrev** [comment]
Approved!

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { useState } from "react";
+
+function App() {
+  const [count, setCount] = useState(0);
```
Comment (ivangoudyrev) [inline frontend/src/App.jsx]: I don't like this....

**Review by ivangoudyrev — CHANGES_REQUESTED** [review]

File: frontend/src/App.jsx (hunk 1 above)
Comment (AloofBuddha) [reply frontend/src/App.jsx]: workin on it

Comment (skepticus1) [reply frontend/src/App.jsx]: this works

**Review by ivangoudyrev — APPROVED** [review]

**Pull request merged**